### PR TITLE
Bug 1408597 Added additional URL instructions to Deploying the Registry Console

### PR DESCRIPTION
**Pull request by bmcelvee**
https://bugzilla.redhat.com/show_bug.cgi?id=1408597

### CODE REVIEW

**Comment by bmcelvee** [comment]
@ahardin-rh Please peer review. Thanks!

File: install_config/registry/deploy_registry_existing_clusters.adoc [hunk 1]
```diff
@@ -337,6 +337,12 @@ $ oc new-app -n default --template=registry-console \
     -p COCKPIT_KUBE_URL=$(oc get route registry-console -n default --template='https://{{ .spec.host }}')
 ----
 
+[NOTE]
+====
+If the redirection url is wrong when you are trying to log in to the
```
Comment (ahardin-rh) [inline install_config/registry/deploy_registry_existing_clusters.adoc]: s/url/URL
s/check your oauthclient/check your OAuth client

**Comment by ahardin-rh** [comment]
@bmcelvee Just some minor feedback from me. Other than that, looks good! :star:

**Comment by bmcelvee** [comment]
Thanks @ahardin-rh!

**Comment by bmcelvee** [comment]
[rev_history]
|xref:../install_config/registry/deploy_registry_existing_clusters.adoc#deploying-the-registry-console[Setting up the Registry -> Deploying a Registry on Existing Clusters]
|Added additional URL instructions to the xref:../install_config/registry/deploy_registry_existing_clusters.adoc#deploying-the-registry-console[Deploying the Registry Console] section.
%

**Pull request merged**